sw/device/tests/sim_dv/i2c_device_tx_rx_test.c

```diff
@@ -71,6 +71,13 @@ static volatile const uint8_t kI2cByteCount = 0;
 static volatile bool tx_empty_irq_seen = false;
 static volatile bool cmd_complete_irq_seen = false;
 
+/**
+ * these variables store values based on kI2cIdx
+ */
+static uint32_t i2c_irq_fmt_watermark_id;
+static uint32_t i2c_base_addr;
+static top_earlgrey_plic_irq_id_t plic_irqs[9];
```

Comment:
`9` should probably be replaced by a constant `kNumI2cIrqs` or similar.

Reply:
It doesn't look like you're reading this table.

Reply:
@vogelpi OK, I've handled this by defining a constant `#define NUM_I2C_IRQS 6`

@drewmacrae Indeed, that was an ommision in the previous version. `plic_irqs[]`is now used in the `test_main` function.